generate.py

```diff
@@ -421,6 +430,24 @@ def drawStats(frame):
 		, font
 	)
 
+	# =========================
+	# 3. Render the trt box
+
+	# Player 1 TRT Box
+	trt_box = trt_box_img.copy()
+	draw = ImageDraw.Draw(trt_box)
+	draw.text(trt_box_header_xy, "TRT", (255,255,255), font=font_trt)
+	draw.text(trt_box_value_xy, player1.getTRTLabel(), (255,255,255), font=font_trt)
+	frame.paste(trt_box, player1.trt_stats_xy, trt_box)
+
+	# Player 2 TRT Box
+	trt_box = trt_box_img.copy()
+	draw = ImageDraw.Draw(trt_box)
+	draw.text(trt_box_header_xy, "TRT", (255,255,255), font=font_trt)
```

Comment:
Since we always make a copy of the box `trt_box_img`. The box header "TRT" could be written into it just once at bootstrap time, instead of doing for both players at every change. I'll do that after I merge.

Reply:
I see, you're totally right!  This is a good optimization.  Thanks for catching it!  💯